### PR TITLE
Update to latest Polkadot & Substrate

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: test/parachain/Cargo.toml [hunk 1]
```diff
@@ -35,7 +35,7 @@ sc-transaction-pool = { git = "https://github.com/paritytech/substrate", branch
 sp-transaction-pool = { git = "https://github.com/paritytech/substrate", branch = "cumulus-branch" }
 sc-network = { git = "https://github.com/paritytech/substrate", branch = "cumulus-branch" }
 sc-client = { git = "https://github.com/paritytech/substrate", branch = "cumulus-branch" }
-sc-basic-authorship = { git = "https://github.com/paritytech/substrate", branch = "cumulus-branch" }
+sc-basic-authorship = { git = "https://github.com/paritytech/substrate", branch = "cumulus-branch", version = "0.8.0-alpha.5" }
```
Comment (cecton) [inline test/parachain/Cargo.toml:38]: This looks fishy. Is it intended??

File: test/parachain/runtime/build.rs [hunk 2]
```diff
@@ -19,7 +19,7 @@ use wasm_builder_runner::WasmBuilder;
 fn main() {
 	WasmBuilder::new()
 		.with_current_project()
-		.with_wasm_builder_from_crates("1.0.9")
+		.with_wasm_builder_from_git("https://github.com/paritytech/substrate.git", "0b30207969fdde85e0dad785750757399cd0e3e4")
```
Comment (cecton) [inline test/parachain/runtime/build.rs:22]: There is no branch or tag we can use instead? Like cumulus-branch for instance?

File: test/runtime/build.rs [hunk 3]
```diff
@@ -19,7 +19,7 @@ use wasm_builder_runner::WasmBuilder;
 fn main() {
 	WasmBuilder::new()
 		.with_current_project()
-		.with_wasm_builder_from_crates("1.0.9")
+		.with_wasm_builder_from_git("https://github.com/paritytech/substrate.git", "0b30207969fdde85e0dad785750757399cd0e3e4")
```
Comment (cecton) [inline test/runtime/build.rs:22]: Same

File: test/parachain/Cargo.toml (hunk 1 above)
Comment (bkchr) [reply test/parachain/Cargo.toml:38]: Cargo wants this.

File: test/parachain/runtime/build.rs (hunk 2 above)
Comment (bkchr) [reply test/parachain/runtime/build.rs:22]: A commit is a tag.

Comment (cecton) [reply test/parachain/runtime/build.rs:22]: And why didn't you pin to the branch instead? You needed that commit in particular? It wouldn't have work on cumulus-branch?

**Pull request merged**